test_stack.py

```diff
@@ -99,17 +106,34 @@ def test_dup():
     assert eval_program("push 1; dup; push 2; dup 3") == [1, 1, 2, 1, 1, 2]
     with pytest.raises(IndexError):
         # Cannot duplicate the top element, since there is not top element
-        assert eval_program("dup")
+        eval_program("dup")
     with pytest.raises(IndexError):
-        assert eval_program("push 1; dup 2")
+        eval_program("push 1; dup 2")
 
 
 def test_quiet():
-    expected = [Instr('div', prefix=[]), Instr('add', prefix=['quiet'])]
-    assert list(stack.parse_program('div; quiet add')) == expected
-
     assert eval_program("push 3; push 5; quiet add") == [3, 5, 8]
     assert eval_program("push 3; push 5; quiet mul") == [3, 5, 15]
     assert eval_program("push 3; push 5; quiet sub") == [3, 5, -2]
     assert eval_program("push 3; push 5; quiet div") == [3, 5, 0.6]
     assert eval_program("push 3; push 5; quiet pow") == [3, 5, 3**5]
+
+
+def test_jump():
+    # It should be possible to jump one past the end of the code, but no more
```

Comment:
What should happen if we try jumping father back than there are instructions? For example, should `push -2; jump` raise `IndexError`?